### ISSUE TITLE
Missing file after cloning and attempting to run.

### ISSUE THREAD

**Issue by Striderzet** [Mebius973/LegoIos#32]
Missing this file and variables that are in it:

"......GIT Contributes/LegoIos/LegoIos/AppConfig.swift' (in target 'LegoIos' from project 'LegoIos')"

**Comment by Mebius973** [comment]
This file contains an API access key, that's why it's explicitly ignored in git. If you want to create one, the file is 
`import Foundation
class AppConfig {
    static let LegoApiKey = "API_KEY"
}`

You can obtain an API_KEY by registering at https://rebrickable.com/

**Comment by Mebius973** [comment]
I know, it's not the best way to deal with such things but I didn't get the time to think of something better as I'm currently struggling with tests failing on CI for bad reasons.

**Comment by Striderzet** [comment]
Ok done, it works now. So if need to do pull request for this i just need to get rid of the API key correct, not the whole file? I think that would be easier fo anyone else that may stumble on this and want to contribute.

**Comment by Mebius973** [comment]
I agree, but I thought adding it to .gitignore was a better option to make sure I was not commiting it by mistake. The route cause is actually that such keys should not be dealt with in a dedicated class but be an environment variable or something likewise. I'm currently working on tests. Once it's done I'll try to solve that problem
